src/Compilers/VisualBasic/Test/Symbol/Microsoft.CodeAnalysis.VisualBasic.Symbol.UnitTests.vbproj

```diff
@@ -3,7 +3,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>$(RoslynPortableTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>net472;netcoreapp3.0</TargetFrameworks>
```

Comment:
>net472;netcoreapp3.0 [](start = 22, length = 20)

What values would `$(RoslynPortableTargetFrameworks)` produce? (ie. what's the difference?)

Reply:
> What values would $(RoslynPortableTargetFrameworks) produce? (ie. what's the difference?)

I think it is: "net472;netcoreapp2.1"

---
In reply to: [288298770](https://github.com/dotnet/roslyn/pull/35957#discussion_r288298770) [](ancestors = 288298770)